extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -90,14 +93,35 @@ amp-carousel .amp-carousel-button.amp-disabled {
 
 .-amp-slide-item {
   display: none;
+  flex: 1 0 100% !important;
```

Comment:
Shouldn't it be `0 0 100%`? We don't want it to shrink or grow from basis, right?